server/models/user.js

```diff
@@ -1 +1,71 @@
 // init
+const mongoose = require('mongoose')
+const jwt = require('jsonwebtoken')
+
+const userSchema = new mongoose.Schema({
+    un : {  // username
+        type: String,
+        unique : true,
+        required : true,
+        index : true,
+        alias : 'username'
+    },
+    fn : { // first Name
+        type : String,
+        alias : 'firstName',
+        index : true,
+    },
+    ln : { // last Name
+        type : String,
+        alias : 'lastName'
+    },
+    email : {
+        type : String,
+        required : true,
+        index : true
+    },
+    a : { // avatar
+        type : String,
+        alias : 'avatar'
+    },
+    currentGame : { // current Game details
+        gameLobby : {
+            type : String,
+            trim : true,
+            lowercase : true
+        },
+        currentScore : Number

```

Comment:
Bluff does not have a definitive score in a running game, so this shouldn't be required.

Reply:
Ok